packages/os/bootstrap-containers-toml

```diff
@@ -0,0 +1,16 @@
+[required-extensions]
+bootstrap-containers = "v1"
++++
+{{#if settings.bootstrap-containers}}
+{{#each settings.bootstrap-containers}}
+["{{@key}}"]
+source = "{{this.source}}"
+mode = "{{this.mode}}"
+{{#if this.user-data}}
+user-data = "{{{this.user-data}}}"
+{{/if}}
+{{#if this.essential}}
+essential = {{this.essential}}
+{{/if}}
```

Comment:
Boolean value. Not sure if we can use `{{ default false this.essential }}` - if not then then let's add an `else` block.